util/Migrator/DbScripts/2022-07-11_00_Initial_Organization.sql

```diff
@@ -0,0 +1,547 @@
+IF OBJECT_ID('[dbo].[OrganizationPasswordManager]') IS NOT NULL
```

Comment:
I have quite a few reservations as this script is supposed to be re-runnable, so if this is run, say, 6 months from now in production, it's going to drop and re-create this table and unlike a stored procedure, that spells disaster/data-loss, etc.

Please consider changing this to `IS NULL` instead and moving the `CREATE TABLE` block into the conditional block so it's created only when it doesn't already exist and there is no `DROP TABLE` statement.

Reply:
Excellent point!